src/CMakeLists.txt

```diff
@@ -53,10 +53,29 @@ target_include_directories(ecpint
     $<INSTALL_INTERFACE:$<INSTALL_PREFIX>/include>
 )
 
-target_link_libraries(ecpint
- PRIVATE
-  Faddeeva
-)
+if (LIBECPINT_USE_CERF)
+    find_package(cerf CONFIG REQUIRED) # QUIET)  # COMPONENTS C)
```

Comment:
@loriab  @jwuttke Which version of libcerf comes with a `cerfConfig.cmake`?

Reply:
Got it: https://jugit.fz-juelich.de/mlz/libcerf/-/merge_requests/2

Reply:
I patched the above PR into the Fedora and openSUSE RPM, so we could bump libecpint.